src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs

```diff
@@ -24,5 +24,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```

Comment:
Please do not update the `AssemblyVersion` here

Reply:
I changed this, but it looks like it was already added in another PR: 
 https://github.com/Azure/azure-sdk-for-net/blob/0af3868778c7378b8a49dc7105760affe6a18428/src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs#L27